src/main/groovy/jenkins/automation/builders/BaseJobBuilder.groovy

```diff
@@ -25,46 +26,18 @@ class BaseJobBuilder {
     String description
     List<String> emails
 
-    Job build(DslFactory factory){
-        factory.job(name){
+    Job build(DslFactory factory) {
+        factory.job(name) {
             it.description this.description
-            addBaseStuff(delegate,this.emails)
-        }
-    }
-
-/**
- *  Adds colorized input plugin support
- * @param context
- */
-    static void addColorizeOutput(context){
-
-        context.with{
-            colorizeOutput()
-        }
-    }
-
-/**
- *  Adds a minimum base functionality required -build claiming, notifications and log.
- * @param context  delegate passed in context
- * @param emails  email list of the developers for notification
- */
-    static void addBaseStuff(context, emails) {
-        context.with{
-            wrappers{
-                addColorizeOutput(delegate)
-            }
-            logRotator {
-                //TODO: make it parametrized to builder
-                numToKeep(10)
-            }
-            publishers{
-                allowBrokenBuildClaiming()
-
-                if(emails){
+            CommonUtils.add_defaults(delegate)
```

Comment:
Well... that means that every single builder will also need to support that parameter. Is it worth it, compared with just adding a new block in your jobs.groovy for that job in the cases where you do need to overwrite it?